File: doc/snippets/whatsapp/message/media/sticker/snippet.py
import requests
import json

url = 'https://api.wababox.net/waba/1.1/json/<WABAID>/message/<PHONEID>'
headers = {
    'Authorization': 'App <APIKEY>',
    'Content-Type': 'application/json'
}
data = {
    'recipients': [
        "+336XXXXXXXX"
    ],
    'contentMessage': {
        'type': 'sticker',
        'sticker': {
            'link': 'https://www.example.com/fr/file.webp',
            'filename': 'file.webp'
        }
    }
}

response = requests.post(url, headers=headers, data=json.dumps(data))

print(response.status_code)
print(response.json())

File: doc/snippets/whatsapp/message/reaction/snippet.js
const url = 'https://api.wababox.net/waba/1.1/json/<WABAID>/message/<PHONEID>';
const headers = {
    'Authorization': 'App <APIKEY>',
    'Content-Type': 'application/json'
};
const data = {
    recipients: [
        "+336XXXXXXXX"
    ],
    contentMessage: {
        type: "reaction",
        reaction: {
            message_id: 'wamid.xxxxxxxxxxxxxxxxxxxxxxxxxxxx',
            emoji: "😀"
        }
    }
};

fetch(url, {
    method: 'POST',
    headers: headers,
    body: JSON.stringify(data)
})
    .then(response => response.json())
    .then(data => console.log(data))
    .catch(error => console.error('Error:', error));
